bedrock/base/templates/product-all-macros.html

```diff
@@ -84,7 +84,7 @@ <h4 class="not-found{% if builds %} hide{% endif %}">{{ _('No matching languages
 #}
 {% macro build_link(build, platform, tooltip) %}
   {% if build.platforms[platform] %}
-    <td class="download {{ platform }}"><a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}">{{ _('Download') }}</a></td>
+    <td class="download {{ platform }}"><a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}" data-link-type="download" data-download-os="Desktop">{{ _('Download') }}</a></td>
```

Comment:
This macro is used by multiple platforms and browser versions, including Android and Thunderbird. You may need to add some additional logic to identify the correct value depending on the product/platform. The URL's where this is in use are as follows:

- `/firefox/all/`
- `/firefox/beta/all/`
- `/firefox/developer/all/`
- `/firefox/organizations/all/`
- `/firefox/android/all/`
- `/firefox/android/beta/all/`
- `thunderbird/all/`
- `/thunderbird/beta/all/`
- `/thunderbird/earlybird/all/`